language/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -739,7 +739,10 @@ fn handle_expected_output(test_path: &Path, output: impl AsRef<str>) -> Result<(
     if !exp_path.exists() {
         std::fs::write(&exp_path, "").unwrap();
     }
-    let expected_output = std::fs::read_to_string(&exp_path).unwrap();
+    let expected_output = std::fs::read_to_string(&exp_path)
+        .unwrap()
+        .replace("\r\n", "\n")
+        .replace("\r", "\n");
```

Comment:
this change has been included in #132

Reply:
thanks, this was removed from current pr